docs/source/getting-started-setup.md

```diff
@@ -196,7 +196,7 @@ The ExecuTorch repo uses CMake to build its C++ code. Here, we'll configure it t
   ```bash
   # Clean and configure the CMake build system. Compiled programs will
   # appear in the executorch/cmake-out directory we create here.
-  (rm -rf cmake-out && mkdir cmake-out && cd cmake-out && cmake ..)
+  (rm -rf cmake-out && mkdir cmake-out && cd cmake-out && cmake .. && cd ..)
```

Comment:
This isn't necessary since it's in a subshell (the parentheses). That's why I put this in parens, because it won't affect the current directory of the parent shell